packages/clerk-js/src/core/clerk.ts

```diff
@@ -451,6 +451,9 @@ export default class Clerk implements ClerkInterface {
       params.afterSignUpUrl || params.redirectUrl || displayConfig.afterSignUpUrl,
     );
 
+    const navigateToContinueSignUp = makeNavigate(displayConfig.signUpUrl + '/continue');
```

Comment:
This doesn't look like it will work if the sign up component is mounted with hash routing.

Reply:
@SokratisVidros I've left testing CRA testing for last, but will of course not merge unless it works there as well.

Reply:
Fwiw... you can test hash based routing in any framework. Just drop the component without `routing` and `path` props.

This shouldn't work because it will navigate to `/sign-up/continue`, which routers treat as separate route than `/sign-up`.

If we're redirecting to a nested path and we don't know whether it's `hash` or `path` mounted, we should always use `hash`. Then, If it turns out that <SignUp/> is path-mounted, <SignUp/> itself can strip the hash away.

`navigateToFactorTwo` looks like it always uses hash routing, which is what I'd expect.

Reply:
Btw, I originally routed to `#/continue` but this does not work when we're already in the `SignUp` component.